@FISHDATA/plotDist2centerline.m

```diff
@@ -13,27 +30,50 @@ function plotDist2centerline(obj)
         elseif pIdx(i) == 1
             y2CL(i) = yDiv - heads(i,2);
         end
-
-
     end
 
     idx = abs(y2CL) > 200;
     y2CL(idx) = nan;
     y = (y2CL./(height/2)).*15;
-
-
-    shockTiming = obj.Res.PIshock.ShockTiming / frameRate / 60 + 10;
-
+    shockTiming = obj.Res.PIshock.ShockTiming / frameRate / 60 + 10; 
+  ExtinctTime = obj.Res.ExtinctTime/60+31;
+    if phase==0
+       plotFigure(1,29399,y,shockTiming,ExtinctTime,frameRate,phase)     % new data: 29399->29400
+     elseif phase==1
+       plotFigure(1,6000,y,shockTiming,ExtinctTime,frameRate,phase)
+     elseif phase==2
+       plotFigure(6001,18000,y,shockTiming,ExtinctTime,frameRate,phase)
+     elseif phase==4 
+       plotFigure(18600,29399,y,shockTiming,ExtinctTime,frameRate,phase)   % new data: 29399->29400
```

Comment:
we should be able to deal with exceptions, such as phase == 3, or even phase is not a number.